citi-onboarding/Aqua-Jr
# Hotfix/spaces between sections

# What i did:
- Fix the spaces between the sections
- Fix favicon
- Fix title
# How to test:
- See if the spaces between the sections are good(in others words, if the spaces are look like in the mockuo).
- See if the favicon is pretty(in my particular opinion, it's cute).
- See if appers 'Aqua jr' in the title of the page.
**Extra**: see if everything works like before.
